packages/transactions/src/keys.ts

```diff
@@ -158,26 +171,27 @@ export function createStacksPrivateKey(key: string | Buffer): StacksPrivateKey {
   return { data, compressed };
 }
 
-export function makeRandomPrivKey(entropy?: Buffer): StacksPrivateKey {
-  const ec = new EC('secp256k1');
-  const options = { entropy: entropy || randomBytes(32) };
-  const keyPair = ec.genKeyPair(options);
-  const privateKey = keyPair.getPrivate().toString('hex', 32);
-  return createStacksPrivateKey(privateKey);
+export function makeRandomPrivKey(): StacksPrivateKey {
+  return createStacksPrivateKey(utils.bytesToHex(utils.randomPrivateKey()));
 }
```

Comment:
Perhaps someone else should comment on the change in function signature here -- was the entropy argument ever used by library clients?

Reply:
IIRC it was used for deterministic unit tests at some point.